R-proj/src/rounding.cpp

```diff
@@ -105,8 +107,8 @@ Rcpp::List rounding (Rcpp::Reference P, Rcpp::Nullable<std::string> method = R_N
                 // low dimensional
                 throw Rcpp::exception("volesti supports rounding for full dimensional polytopes. Maybe call function get_full_dimensional_polytope()");
             }
-            HP.normalize();
             InnerBall = HP.ComputeInnerBall();
+            if (InnerBall.second < 0.0) throw Rcpp::exception("Unable to compute a feasible point.");
             if (method_rcpp.compare(std::string("max_ellipsoid")) == 0) {
                 round_res = max_inscribed_ellipsoid_rounding<MT, VT>(HP, InnerBall);
```

Comment:
Why InnerBall as input? The mentod needs an interior point

Reply:
Yes I have changed it. Thanks.